src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/StringBinding.java

```diff
@@ -50,6 +51,7 @@
         return new SingleEnvironment(getCredentials(build).getSecret().getPlainText());
     }
 
+    @Symbol("string")
```

Comment:
:bug:  Does "string" immediately say credentials to users?  I feel like this will bite us -- maybe stringCredential instead?  Or stringCredentialBinding?